src/test/java/com/rybka/service/connector/ExchangeRateConnectorTest.java

```diff
@@ -0,0 +1,45 @@
+package com.rybka.service.connector;
```

Comment:
Can't find tests for PrimeExchangeRateConnector

Reply:
fixed